packages/block-library/src/post-date/block.json

```diff
@@ -3,7 +3,7 @@
 	"apiVersion": 3,
 	"name": "core/post-date",
 	"title": "Date",
-	"category": "theme",
```

Comment:
Many of these blocks have been switched from "theme" to "design" category, but from what I can tell from the issue, these blocks are in a subcategory within "theme". 

![Screenshot 2024-07-31 at 12 21 55 PM](https://github.com/user-attachments/assets/aab3d8fb-1115-46f7-b341-a7f6778a9fa3)

Am I looking at the wrong reference for what the categories should be?

Reply:
No, you're right. I misread the doc and renamed the theme category; undone as of the latest commit.
I'm still wondering what the best way is to introduce the concept of a subcategory. [In the discussion](https://github.com/WordPress/gutenberg/issues/53431), I raised the idea of using the keyword attribute, but I'm not sold yet.